src/directory-sync/interfaces/list-directories-options.interface.ts

```diff
@@ -2,5 +2,6 @@ import { PaginationOptions } from '../../common/interfaces/pagination-options.in
 
 export interface ListDirectoriesOptions extends PaginationOptions {
   domain?: string;
+  organization_id: string;
```

Comment:
This should be optional:

```suggestion
  organization_id?: string;
```